app/api/users/route.ts

```diff
@@ -1,23 +1,26 @@
 import { NextRequest, NextResponse } from "next/server";
-import UserModel from "@/utils/models/UserModel";
-import connectDB from "@/utils/config/db";
 import axios from "axios";
+import { supabaseBrowser } from "@/utils/supabase/client";
+
+const supabase = supabaseBrowser();
 
 export async function POST(request: NextRequest) {
   try {
-    console.log(request);
     const { git_username } = await request.json();
 
     if (!git_username) {
       return new Response("Username is required", { status: 400 });
     }
 
-    await connectDB();
-    const existingUser = await UserModel.findOne({ git_username });
+    const existingUser = await supabase
+      .from("recent_users")
+      .select("*")
+      .eq("username", git_username)
+      .single();
 
-    if (existingUser) {
-      console.log("user already exits");
-      return new Response("Username updated successfully", { status: 200 });
+    if (existingUser.data && existingUser.data.name) {
+      console.log("User already exists");
```

Comment:
remove consoles and display data in reverse order as we need recent user first

Reply:
> remove consoles and display data in reverse order as we need recent user first

Hey @ashutosh-rath02, the data is displayed in the reverse order, meansing new user at first.
```js
// /app/api/users/route.ts 
const { data: users, error } = await supabase
      .from("recent_users")
      .select("*")
      .order("id", { ascending: false })
      .limit(7);
```
we have kept `ascending: false` which returns the latest user first.